### PR TITLE
Config generation now only needs the project ID

### LINKED ISSUE #211: Make "yarn setup" read the firebase web snippet
Make it read and convert the snippet to generate the `.env.local` config file from it.

### PR DESCRIPTION
**Pull request by rubenspgcavalcante**
Closes #211

### CODE REVIEW

**Comment by rubenspgcavalcante** [comment]
Ops, I need to update the README. I'll do another commit :)

File: README.md [hunk 1]
```diff
@@ -29,17 +29,20 @@ We are always looking for help. Working on this project is an opportunity to use
 
     - Go to https://console.firebase.google.com/ and click "Add project";
     - Enter "Boxwise Development" into the name field and click "Create Project";
-    - Once that has completed, click "Add Firebase to your web app". Ignore the prompt to copy and paste the entire snippet, and look at the fields in the `config` object;
-    - Create a new local config running `yarn setup`, filling with the Firebase project ID (you may be asked to login). This will create a `.env.local` file;
+    - Once that has completed, click "Add Firebase to your web app". Ignore the prompt to copy and paste the snippet for now;
```
Comment (bfirsh) [inline README.md]: This step can be removed entirely - all it does is open the modal with the config details, I think.

File: README.md [hunk 2]
```diff
@@ -29,17 +29,20 @@ We are always looking for help. Working on this project is an opportunity to use
...
     - Click "Database" in the left hand menu, click "Get Started" underneath Cloud Firestore, then click "Enable";
     - Click "Authentication" in the left hand menu, then the "Sign-in method" tab, then click "Email/Password", flip the first "Enable" switch, then click "Save".
 
-4.  Log into Firebase, then select the app you created in the previous step. Give it the alias "development".
+4. Create a new local config running `yarn setup`, filling with the Firebase project ID (you may be asked to login). This will create a `.env.local` file.
+    - **How to get the Firebase project ID:** Click on the settings gear next to "Project Overview". A dropdown menu will be shown, click on "Project Configuration".
+    The project ID will be listed on first card (Your project). Will be the item right over the API key.
+
+5.  Log into Firebase, then select the app you created in the previous step. Give it the alias "development".
```
Comment (bfirsh) [inline README.md]: Yarn setup could do this for us...

**Comment by bfirsh** [comment]
One minor problem but I’ll merge and fix later. :)

**Pull request merged**